doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -690,6 +690,11 @@ Modem libraries
     * Convenience function to get :c:struct:`location_data_details` from the :c:struct:`location_event_data`.
     * Location data details for event :c:enum:`LOCATION_EVT_RESULT_UNKNOWN`.
 
+* :ref:`lte_lc_readme` library:
+
+  * Removed ``AT%XRAI`` related deprecated functions ``lte_lc_rai_param_set()`` and ``lte_lc_rai_req()``, and Kconfig option :kconfig:option:`CONFIG_LTE_RAI_REQ_VALUE`.
+    The application uses the Kconfig option :kconfig:option:`CONFIG_LTE_RAI_REQ` and ``SO_RAI`` socket options instead.
```

Comment:
So do we add something about all removed and deprecated APIs into the migration guide? And should it be when deprecated or removed?
@peknis @gregersrygg

Reply:
When removed, at the latest. If the deprecated options and functions are used frequently, it might also be a good idea to indicate that in the migration guide.

Reply:
I don't expect that these would be used too much. Also, I don't know if we would say much more in the migration guide than what is written here.